File: src/main/java/com/exsoloscript/mysteryteams/util/PlayerUtils.java
package com.exsoloscript.mysteryteams.util;

import org.bukkit.Bukkit;
import org.bukkit.entity.Player;

import java.util.ArrayList;
import java.util.List;

public class PlayerUtils {

    public static List<Player> getOnlinePlayersWithoutOps() {
        List<Player> players = new ArrayList<>();
        for (Player p : Bukkit.getOnlinePlayers()) {
            if (!p.isOp())
                players.add(p);
        }

        return players;
    }
}

File: src/main/kotlin/com/exsoloscript/mysteryteams/util/PlayerUtils.kt
package com.exsoloscript.mysteryteams.util

import org.bukkit.Bukkit
import org.bukkit.entity.Player

import java.util.ArrayList

object PlayerUtils {

    val onlinePlayersWithoutOps: List<Player>
        get() {
            val players = ArrayList<Player>()
            for (p in Bukkit.getOnlinePlayers()) {
                if (!p.isOp)
                    players.add(p)
            }

            return players
        }
}
